scripts/st2bootstrap-deb.sh

```diff
@@ -339,17 +339,17 @@ configure_st2_user () {
     sudo useradd stanley
   fi
 
-  sudo mkdir -p /home/stanley/.ssh
+  sudo mkdir -p ~stanley/.ssh
```

Comment:
Change like that will require `st2.conf` adjustments as well: https://github.com/StackStorm/st2/blob/master/conf/st2.package.conf#L73
And not sure if that's a good idea. Or we stick with `~` everywhere or with old `/home/stanley/` as it was before.